<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicPlayModule</class>
 <widget class="QWidget" name="MusicPlayModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MusicPlayModule</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgba(0, 170, 255, 150)</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>110</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <widget class="QSlider" name="musicSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>280</width>
      <height>4</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::add-page:Horizontal
{     
background-color: rgba(255, 255, 255,150);
height:4px;  
} 
QSlider::sub-page:Horizontal   
{  
background-color: rgb(255, 255, 255);
height:4px;
}
QSlider::groove:Horizontal   
{ 
background:transparent; 
height:4px;
}</string>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="musicName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background:none;</string>
    </property>
    <property name="text">
     <string>桃几OvO - Letter song.flac</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>70</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:none;
color:white;</string>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
